gpii/node_modules/processReporter/src/processesBridge.js

```diff
@@ -0,0 +1,236 @@
+/*!
+GPII Process Reporter processes bridge -- gpii.processes.
+
+Copyright 2014 Inclusive Design Research Centre, OCAD University
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/gpii/universal/LICENSE.txt
+*/
+
+/*global require */
+
+"use strict";
+
+var fluid = require("infusion");
+var gpii = fluid.registerNamespace("gpii");
+
+gpii.processes = fluid.registerNamespace("gpii.processes");
+
+fluid.defaults("gpii.processes", {
```

Comment:
I recommend browser-based tests for these since this self-contained functionality requires no platform support